### DISCUSSION TITLE
Change at times of day?

### DISCUSSION THREAD

**Discussion by PlutoMarionette** [rocksdanister/lively#1208] [Ideas]
I might just be very, very stupid if this is already a feature, but if at designated times, the wallpaper changes from one to another, I think it'd be pretty cool. Specifically if you want your wallpaper to mirror a day and night sort of animation. For context I have two different images/wallpapers that look the same with the only difference being one is night and the other is day. If past a certain point in the day, the second wallpaper fades in/replaces the other that could (theoretically) mirror a night and day cycle. Then again, I might just be dumb and this is all ready a feature, in which case, sorry for wasting time-I wold also like to add that I only know bare bones on software capabilities so I'm pretty sure this isn't possible, but it was worth a shot anyway

**Comment by MarsVertigo** [comment]
I don't think there is such function implemented yet.

**Comment by rocksdanister** [comment]
Possible https://github.com/rocksdanister/lively/issues/137 ?
